site/content/docs/5.0/customize/sass.md

```diff
@@ -6,7 +6,7 @@ group: customize
 toc: true
 ---
 
-Utilize our source Sass files to take advantage of variables, maps, mixins, and more. In our build we've increased the Sass rounding precision to 6 (by default it's 5) to prevent issues with browser rounding.
+Utilize our source Sass files to take advantage of variables, maps, mixins, and more. In our build we've left Dart Sass's rounding precision of 10. We recommend maintaining a precision of at least 6 to prevent issues with browser rounding.
```

Comment:
We might need to reword this @mdo. Dart Sass does not offer the ability to customize the precision anyway.

Reply:
I wanted to keep the mention of a minimum of 6 since it's caused issues in the past, but perhaps we should specify that lowering it can only be done through further processing, such as minification? I was trying to keep it terse.

Reply:
Should it also specify that Dart Sass's precision is locked at 10?